networkx/algorithms/centrality/betweenness.py

```diff
@@ -135,11 +143,14 @@ def betweenness_centrality(
     Zero edge weights can produce an infinite number of equal length
     paths between pairs of nodes.
 
-    The total number of paths between source and target is counted
-    differently for directed and undirected graphs. Directed paths
-    are easy to count. Undirected paths are tricky: should a path
-    from ``u`` to ``v`` count as 1 undirected path or as 2 directed paths?
-    For this implementation of `betweenness_centrality`, we do the latter.
+    Directed graphs and undirected graphs are handled differently.
+    In directed graphs, each pair of nodes is considered once in each direction,
+    as the shortest paths can change.
```

Comment:
"are handled differently" -> "count paths differently"
"pair of nodes" -> "pair of source-target nodes".   "once" -> "separately"